hub/apps/get-started/uno-setup.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Uno Setup Guide
+description: Learn how to reach users on other platforms like Web, iOS, Android, and Linux with minimal changes to the C#/WinUI 3 simple photo viewer built in the previous tutorial. We'll use Uno Platform to create a new multi-platform app, which we can move code from the existing desktop project to.
+ms.topic: article
+ms.date: 05/21/2023
+keywords: Windows, App, SDK, WinUI 3, WinUI, photo, viewer, Windows 11, Windows 10, XAML, C#, uno platform, uno
+ms.author: aashcraft
+author: alvinashcraft
+ms.localizationpriority: medium
+---
+
+## Finalize your environment
```

Comment:
```suggestion
# Finalize your environment
```

Reply:
If the content of this uno-setup file is intended to only be used inline as an include file in other markdown pages, then this heading should remain an H2 (despite the build warning).

Reply:
Thanks for the confirmation @alvinashcraft